src/examples/deploy_with_setup.yml

```diff
@@ -0,0 +1,14 @@
+description: |
+  A workflow to deploy your project using waypoint, with setup steps.
+usage:
+  version: 2.1
+  orbs:
+    waypoing: circleci/waypoint@x.y # This version number refers to the version of the orb, not the version of NodeJS
```

Comment:
"waypoing" > "waypoint".

Do we also want to leave x.y?

Reply:
Definitely hearkens back to a larger debate about orb versioning in examples. I'm leaning more towards leaving it as `x.y` until we have a better solution (at the registry level?).